modules/libcom/RTEMS/posix/rtems_init.c

```diff
@@ -1018,12 +1061,12 @@ POSIX_Init ( void *argument __attribute__((unused)))
      * -net nic,model=e1000 -net user,restrict=yes \
      * -append "--video=off --console=/dev/com1" -kernel libComTestHarness
      */
-    printf("\n***** Initializing network (libbsd, dhcpcd) *****\n");
-    rtems_bsd_setlogpriority("debug");
+    printf("\n***** Initializing network (libbsd) *****\n");
+    if(0) rtems_bsd_setlogpriority("debug");
```

Comment:
Is this PR intended for immediate consideration, or should it be marked as a draft?

Reply:
I would love it if that was seen as a "serious" next step and could be incorporated into the next release.

I am now using rtems6 with 6-freebsd-12. This gives support for NFSv4 which had been requested by some.
I am also now using a nexus ethernet driver with the MVME6100.

Beside the dhcpcd support the CPU can now also get its network configuration in the "old" way from the NVRAM. That was also requested by a few places. 

Also the VMEbus support is now back with rtems6 (I had this still excluded since still tests were missing from my side).

Reply:
I'm referring specifically to `if(0)`, which imo. is unsightly clutter.

Making the log level/priority configurable would be an improvement, with NVRAM storage being an obvious place to store such an option.  Though, I think that "debug" should remain the default so that new users have more indication of what is going on.